### PR TITLE
BTFS-1214

### PR DESCRIPTION
**Pull request by NathanQiu666**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior?** (You can also refer to a JIRA ticket here)

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **What dependencies / modules need to be updated as pre-requisites?** (Include relevant PR links here)

* **Description of changes**

---

* **Please check if the PR fulfills these requirements**
- [ ] The subject of this PR contains a JIRA ticket BTFS-xxxx (N/A for community PRs)
- [ ] PR description and commit messages are [good and meaningful](https://chris.beams.io/posts/git-commit/)
- [ ] Unit tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **Please make sure the following procedures have been applied before requesting reviewers**
- [ ] Code changes closely follow [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
- [ ] Code has been *rebased* against latest master (`git merge` not recommended, unless you know what you are doing)
- [ ] Code changes have run through `go fmt`
- [ ] Code changes have run through `go mod tidy`
- [ ] All unit tests passed locally (`make test_go_test`)

### CODE REVIEW

File: core/commands/store/path/path.go [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+	Helptext: cmds.HelpText{
+		Tagline: "Modify the Host storage folder path for BTFS client.",
+		ShortDescription: `
+The default local repository path is located at ~/.btfs folder, in order to
+improve the hard disk space usage, provide the function to change the original 
+storage location, a specified path as a parameter need to be passed.
+`,
+	},
+	Arguments: []cmds.Argument{
+		cmds.StringArg("path_name", true, true, "New BTFS Path.").EnableStdin(),
```
Comment (taiyangc) [inline core/commands/store/path/path.go:41]: Use "path-name" instead of "path_name"

File: core/commands/store/path/path.go [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+			propertiesPath = filepath.Join(home, "btfs")
+		} else {
+			propertiesPath = filepath.Join(home, "btfs/bin")
+		}
+	}
+
+	if CheckExist(propertiesPath) == false {
+		err := os.MkdirAll(propertiesPath, os.ModePerm)
+		if err != nil {
+			log.Fatal(err)
```
Comment (taiyangc) [inline core/commands/store/path/path.go:184]: Should not fatal. Even if propertiesPath does not exist, the default ~/.btfs still works. Can log a more detailed error message here.

Comment (NathanQiu666) [reply core/commands/store/path/path.go:184]: yes, agree, the error will not cause current program to exit.

File: core/commands/store/path/path.go (hunk 1 above)
Comment (NathanQiu666) [reply core/commands/store/path/path.go:41]: thanks, name changed

**Pull request closed without merging**